test/javascript/components/common/CopyToClipboardButton.test.js

```diff
@@ -57,13 +56,3 @@ test('copies text to clipboard when pressing Ctrl+C', async () => {
 //     timeout: 2500,
 //   })
 // })
-
-test('hides component if clipboard API is unavailable', async () => {
```

Comment:
@ErikSchierboom I don't see this being part of our implementation in the component anymore. That's why I decided to delete the test. Did I miss anything?

Reply:
I also don't see it anymore, so removing this seems to be fine.